src/TurbulenceClosures/turbulence_closure_implementations/scale_invariant_smagorinsky.jl

```diff
@@ -292,11 +292,25 @@ Base.show(io::IO, closure::ScaleInvariantSmagorinsky) = print(io, summary(closur
 ##### For closures that only require an eddy viscosity νₑ field.
 #####
 
+directionally_averaged_field(grid, ::Val{DirectionalAveraging(1)})         = Field{Nothing, Center,  Center }(grid)
+directionally_averaged_field(grid, ::Val{DirectionalAveraging(2)})         = Field{Center,  Nothing, Center }(grid)
+directionally_averaged_field(grid, ::Val{DirectionalAveraging(3)})         = Field{Center,  Center,  Nothing}(grid)
+directionally_averaged_field(grid, ::Val{DirectionalAveraging((1, 2))})    = Field{Nothing, Nothing, Center }(grid)
+directionally_averaged_field(grid, ::Val{DirectionalAveraging((1, 3))})    = Field{Nothing, Center,  Nothing}(grid)
+directionally_averaged_field(grid, ::Val{DirectionalAveraging((2, 3))})    = Field{Center,  Nothing, Nothing}(grid)
+directionally_averaged_field(grid, ::Val{DirectionalAveraging((1, 2, 3))}) = Field{Nothing, Nothing, Nothing}(grid)
+directionally_averaged_field(grid, ::Val{DirectionalAveraging(Colon())})   = Field{Nothing, Nothing, Nothing}(grid)
+directionally_averaged_field(grid, ::Any)                                  = Field{Center,  Center,  Center}(grid)
+
+
 function DiffusivityFields(grid, tracer_names, bcs, closure::ScaleInvariantSmagorinsky)
 
     default_eddy_viscosity_bcs = (; νₑ = FieldBoundaryConditions(grid, (Center, Center, Center)))
     bcs = merge(default_eddy_viscosity_bcs, bcs)
     νₑ = CenterField(grid, boundary_conditions=bcs.νₑ)
-    #return (; νₑ, LM_avg, MM_avg)
-    return (; νₑ)
+
+    LM_avg = directionally_averaged_field(grid, Val(closure.averaging))
+    MM_avg = directionally_averaged_field(grid, Val(closure.averaging))
```

Comment:
I would use something like this to make things a little simpler:

```julia
# TODO: add `velocities` as argument to `DiffusivityFields` to simplify this
MM_avg = Field(Average(νₑ, dims=closure.averaging.dims))
```

Reply:
but it doesn't matter

Reply:
Yeah I see your point. Since the important part is just to allocate memory, that already does the trick. That does kind of imply that `MM_avg` will later get the average of `νₑ`, which may confuse readers of the code.

How about 

```julia
MM_avg = Field(Average(Field{Center, Center, Center}(grid), dims=closure.averaging.dims))
```
?

Reply:
That's fine, my suggestion merely aimed to avoid allocating the additional 3D temporary array. 

Hopefully this code is temporary. It's just a vehicle for you to validate this closure so it can be used as soon as possible.

Reply:
You can also use an empty / placeholder operation. But don't worry too much let's move on to validation...